source/HL_sys_link.cmd

```diff
@@ -46,6 +46,7 @@
 
 /* USER CODE BEGIN (1) */
 #ifdef DO_NOT_COMPILE
+#error "The error was up here!'

```

Comment:
What error? Can this be more informative?

Reply:
Not sure. I didn't add this. @Taylorr82? @tganley?

Reply:
Oh. I remember. This was added by robert when we were testing the bootloader.